src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public enum Colors {
```

Comment:
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names
it should be Color